backend/Piece.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+
+
+class Turn(Enum):
+    BLACK = 1
+    WHITE = 2
+
+    @classmethod
+    def swap_turn(self, curr):
+        if curr == Turn.BLACK:
+            return Turn.WHITE
+        else:
+            return Turn.BLACK
+
+class Piece:
+    def __init__(self, color):
+        self.color = color
+        self.position = (-1, -1)
+    
+    def get_turn(self):
```

Comment:
This doesnt feel like it belongs in this class. Unless its `get_color`. I think this should be called `get_color`.

Reply:
It was supposed to be -- I swapped the name of the variable after the I made the Turn class. Must've forgotten to swap the function names.